index.html

```diff
@@ -10,20 +10,22 @@
 	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
 	<title>Home Page</title>
 </head>
-<header>
+<header>	
 </header>
 
 <body>
 	<div class="body-menu">
 		<div class="sidebar">
+			<img src="images/logo.jpeg" width="100%">
```

Comment:
talvez seja interessante colocar texto alternativo nas imagens, ajuda a tornar o site mais acessível:

http://blog.handtalk.me/texto-alternativo-alt/

por exemplo

```suggestion
			<img src="images/logo.jpeg" width="100% alt="logotipo da Aceleradora Ágil">
```